plasma/plasma-mvp/morevp-simplified.md

```diff
@@ -0,0 +1,87 @@
+# More Viable Plasma
+
+## TL;DR
+
+This document provides a largely simplified specification for More Viable Plasma, a version of [Minimal Viable Plasma](https://ethresear.ch/t/minimal-viable-plasma/426) (“Plasma MVP”) that removes the need for confirmation signatures.
+
+## Background
+
+Transactions in Plasma MVP are processed in time-wise order.
+The “exit priority” of any UTXO is determined by the block in which the transaction was included, then the index of the transaction within that block, and finally the index of the output in the list of transaction outputs (“output-age priority”).
+Plasma MVP requires the use of confirmation signatures to prevent the operator from including a valid transaction time-wise after an invalid transaction.
+Without confirmation signatures, the Plasma MVP exit priority mechanism would process the invalid exit before the valid exit. 
+
+Confirmation signatures are generally bad user experience.
+Users must sign a transaction, wait for it to be included in a block, and sign a second signature attesting that the transaction was included in a valid block.
+Although clients with “hot” wallets can quickly (and usually automatically) sign these secondary signatures, clients with “cold” wallets will face an annoying user-flow (e.g. clicking some button on a hardware wallet).  
+
+Confirmation signatures must also be published to the Plasma chain to avoid certain attack vectors.
+This significantly reduces the amount of block space available for transactions. 
+
+This motivates the search for an alternative exit game that removes the need for confirmation signatures and simultaneously ensures all valid UTXOs can be processed before all invalid UTXOs. 
+
+## More Viable Plasma
+
+More Viable Plasma (“MoreVP”) proposes an exit game that removes the need for confirmation signatures and ensures correct transaction processing order.
```

Comment:
on the other hand, this description misses the essential setting of the exit game - tx is in-flight.

Reply:
The point here is to give the intuition that the exit game is safe in all contexts, whether the transaction is in-flight or not.

Reply:
I get that, and that's a worthwhile statement to include! If you don't mention in-flightness at all, you're not making a case for the complicated protocol. Let me rephrase:

> essential setting of the exit game - tx **might be** in-flight.

Reply:
Agreed, changed.